regolith/builders/reimbursementbuilder.py

```diff
@@ -118,9 +109,9 @@ def excel(self):
                             se_column = 14
                         dates.append(expdates.get("date"))
                         item_ws.cell(row=r, column=2, value=i)
-                        item_ws.cell(row=r, column=3, value=mdy(expdates.get("date").month,
-                                              expdates.get("date").day,
-                                              expdates.get("date").year))
+                        item_ws.cell(row=r, column=3, value=dt.datetime(int(expdates.get("date").year),
```

Comment:
doesn't 
```
dt.datetime(expdates.get("date)).strftime('%x')
```
work?

Reply:
also I see the same mdy function being used in coa builder.  Please could you do the same cleaning there as well?  Thx.

Reply:
I think the mdy function was defined there, but wasn't used. Thus, I just deleted it.

Reply:
cool.  Remove WIP and request merge if you are ready.  Make sure there is a news.